.azure/pipelines/jobs/default-build.yml

```diff
@@ -95,14 +95,14 @@ jobs:
         vmImage: macOS-10.14
       ${{ if eq(parameters.agentOs, 'Linux') }}:
         ${{ if eq(parameters.useHostedUbuntu, true) }}:
-          vmImage: ubuntu-16.04
+          vmImage: ubuntu-18.04
         ${{ if eq(parameters.useHostedUbuntu, false) }}:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
             name: NetCorePublic-Pool
-            queue: BuildPool.Ubuntu.1604.Amd64.Open
+            queue: BuildPool.Ubuntu.1804.Amd64.Open
```

Comment:
Need to wait until this queue gets rolled out before this will work

Reply:
I used `Ubuntu.1804.Amd64.Open` yesterday, and this morning it worked

Reply:
Really? FR folks claimed it was blocked on https://github.com/dotnet/core-eng/issues/12744

Reply:
There's a big difference between `BuildPool.Ubuntu.1804.Amd64.Open` and `Ubuntu.1804.Amd64.Open`. The second is a Helix queue intended only for testing. Which did you use yesterday @sebastienros❔

@wtgodbe thoughts on undoing the queue change here and getting the rest in while the getting is good❔

Reply:
I used `Ubuntu.1804.Amd64.Open` which was the reason it was cancelled I assume now, and Will reverted this change.